arm/Microsoft.Network/privateEndpoints/deploy.bicep

```diff
@@ -91,6 +77,14 @@ module privateEndpoint_rbac '.bicep/nested_rbac.bicep' = [for (roleAssignment, i
   }
 }]
 
+module privateEndpoint_privateDnsZoneGroups 'privateDnsZoneGroups/deploy.bicep' = [for (privateDnsZoneGroup, index) in privateDnsZoneGroups: {
```

Comment:
More a question: shoul we keep child modules close to the parents and have parent providers after them?

Reply:
Put it closer to the parent object. Just though that children would be after describing the whole main component, i.e. rbac, diag and locks as they are scoped to the parent resource.